### DISCUSSION TITLE
Image Issue

### DISCUSSION THREAD

**Discussion by Chandan-devs-tech** [community/community#40528] [Projects and Issues]
### Select Topic Area

General

### Body

I am not able to view the Image
<img width="960" alt="Screenshot 2022-12-01 11 25 43" src="https://user-images.githubusercontent.com/65088027/205007862-35d2547c-6a6a-481b-9cfb-80d084602a79.png">
 logo in my README.md file, I don't know why it's happening in my repository.

**Comment by jfullstackdev** [comment]
no problem so far on my end, try reloading it

![image](https://user-images.githubusercontent.com/47092464/205076118-7834b316-4ebf-481a-b886-093753cb0bca.png)

but it happens to my SVGs from time to time

if you have extensions, cached data it can affect too, even low computer memory, try closing all windows, log in in Incognito and see what will happen

**Comment by Aman0509** [comment]
I am also facing the same issue. None of the images (present in few repo) opening for me.
Note - Images have png extensions.

![image](https://user-images.githubusercontent.com/56759061/207356855-6f2589c0-5107-44ab-94ed-d12432f98599.png)
